Remotion/Validation/UnitTests/RoleCollectors/RemovingObjectValidatorRegistrationTest.cs

```diff
@@ -72,7 +73,7 @@ public void Initialization_WithCollectorTypeToRemoveFromIsNull ()
     [Test]
     public void Initialization_WithValidatorPredicateIsNull ()
     {
-      var validatorType = typeof(NotEqualValidator);
+      var validatorType = typeof(FakeCustomerValidator);
```

Comment:
this change is correct because we require an IObjectValidator, not an IPropertyValidator.